src/app/operators/components/operator/operator.component.html

```diff
@@ -21,12 +21,18 @@ <h3 class="short-description" [innerHTML]="shortDescription"></h3>
     [operatorExamples]="examples" class="margin-bottom-16">
   </app-operator-examples>
   <app-operator-parameters
-    [operatorParameters]="parameters">
+    [operatorParameters]="parameters"
+    *ngIf="paramaters?.length">
```

Comment:
Typo `parameters`

Reply:
Oops! Thanks for catching, sending up a fix now!